### PR TITLE
Add tags resource

### PR DESCRIPTION
**Pull request by DevProsanta**

### CODE REVIEW

File: src/WordPress.php [hunk 1]
```diff
@@ -12,6 +13,7 @@
 /**
  * @method Comments comments()
  * @method Categories categories()
+ * @method tags tags()
```
Comment (rickwest) [inline src/WordPress.php]: Just one tiny comment - could you uppercase the class name here

**Review by rickwest — CHANGES_REQUESTED** [review]
Hey, thanks for the pull request! I've just left one tiny comment but other than that it looks good to go! Thank you 🙏🏻

**Comment by NazimNasser** [comment]
@rickwest can we please merge this?

**Comment by rickwest** [comment]
Thats for the contribution @DevProsanta

**Pull request merged**